File: Cryptographer's Conundrum/conundrum.php
<?php
/**
 * Problem: Cryptographer's Conundrum
 * https://open.kattis.com/problems/conundrum
 *
 * @author   Martin Björling
 * @email    martinbjorling@gmail.com
 * @version  1.0
 * @since    2015-11-24
 */

$inputString = stream_get_line(STDIN, 1024, PHP_EOL);

$counter = 0;

for ($i = 0; $i < strlen($inputString); $i++) {
	$c = $inputString[$i];

	$j = $i % 3;

	switch ($j) {
		case 0:
			if ($c != "P") $counter++;
			break;
		case 1:
			if ($c != "E") $counter++;
			break;
		case 2:
			if ($c != "R") $counter++;
			break;
		default:
			break;
	}
}

fwrite(STDOUT, "".$counter);

?>

File: Cryptographer's Conundrum/Conundrum.java
/**
 * Problem: Cryptographer's Conundrum
 * https://open.kattis.com/problems/conundrum
 *
 * @author   Martin Björling
 * @email    martinbjorling@gmail.com
 * @version  1.0
 * @since    2015-11-24
 */

import java.lang.Math;
import java.util.Scanner;

public class Conundrum {
	
	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		String inputString = sc.nextLine();

		int counter = 0;

		for (int i = 0; i < inputString.length(); i++) {
			char c = inputString.charAt(i);

			int j = i % 3;

			switch (j) {
				case 0:
					if (c != 'P') counter++;
					break;
				case 1:
					if (c != 'E') counter++;
					break;
				case 2:
					if (c != 'R') counter++;
					break;
				default:
					break;
			}
		}

		System.out.println(Integer.toString(counter));
	}
}